migrations/scripts/2_deploy_board.js

```diff
@@ -0,0 +1,104 @@
+const realm = process.env.WITNET_EVM_REALM || "default"
+const addresses = require("../addresses")[realm]
+const settings = require("../settings")
+
+let WitnetProxy, WitnetRequestBoard
+
+module.exports = async function (deployer, network, accounts) {
+  try {
+    WitnetProxy = artifacts.require(settings.artifacts[realm].WitnetProxy || settings.artifacts.default.WitnetProxy)
+    WitnetRequestBoard = artifacts.require(
+      settings.artifacts[realm].WitnetRequestBoard || settings.artifacts[realm].WitnetRequestBoard
+    )
+  } catch {
+    console.log("Skipped: 'WitnetRequestBoard' artifact not found.")
+    return
+  }
+
+  let deployWRB = true
+  let upgradeProxy = true
+
+  network = network.split("-")[0]
+  if (network in addresses) {
+    if (addresses[network].WitnetRequestBoard) {
+      WitnetRequestBoard.address = addresses[network].WitnetRequestBoard
+      if (!isNullAddress(WitnetRequestBoard.address)) {
+        deployWRB = false
+        if (WitnetProxy.isDeployed() && !isNullAddress(WitnetProxy.address)) {
+          const proxy = await WitnetProxy.deployed()
+          const currentWRB = await proxy.implementation.call()
+          if (currentWRB.toLowerCase() !== WitnetRequestBoard.address.toLowerCase()) {
+            console.log("Info: Witnet proxy implementation mismatch!")
+            console.log()
+            console.log(`  >> WitnetRequestBoard address in file: ${WitnetRequestBoard.address}`)
+            console.log(`  >> WitnetProxy actual WRB instance:    ${currentWRB}`)
+            console.log()
+          } else {
+            upgradeProxy = false
+          }
+        } else {
+          console.log("Fatal: WitnetProxy not deployed?")
+          process.exit(-1)
```

Comment:
This `-1` speaks too loud about your Windows background, in the UNIX world this would totally be `1` :stuck_out_tongue:  

https://tldp.org/LDP/abs/html/exitcodes.html

Reply:
Totally right ! xD Thanks for the docs ref, it will be handy :-)